src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -658,19 +679,25 @@ public void AddMetadata(string section, IDictionary<string,object> metadata)
             {
                 return;
             }
+            var disposableContainer = new DisposableContainer();
```

Comment:
if you make the container `IDisposable` as Jason suggested you'll be able to use `using` here.